### PR TITLE
Feature/note 002

### PR DESCRIPTION
**Pull request by agape94**

### CODE REVIEW

File: app/src/main/java/at/tugraz/ist/swe/note/NoteActivity.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+import android.os.Bundle;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.EditText;
+import android.widget.Toast;
+
+public class NoteActivity extends AppCompatActivity {
+
+    private Menu _menu;
```
Comment (awgrass) [inline app/src/main/java/at/tugraz/ist/swe/note/NoteActivity.java:12]: You are not consistent in your style. Why are you using the underline in front of the variable name? 
All four fields are private (although the two without a modifier are package-private)
We said we gonna use the Android Code Style conventions. 
None of these fields is declared in the right way, as you can see in https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions

**Review by awgrass — COMMENTED** [review]
Did you really merge develop into this branch?

**Comment by haubitzer** [comment]
@wigglepee yes and no, GitHub did it because I fixed the conflict

**Comment by awgrass** [comment]
> @wigglepee yes and no, GitHub did it because I fixed the conflict

How did you fix the conflict?

**Pull request closed without merging**